.github/pull_request_template.md

```diff
@@ -26,10 +26,12 @@ Not applicable.
 
 
 
-# Checklist
-
-- [ ] l10n changes have been submitted to the l10n repository, if any.
+# Checklist (Definition of Done)
+- [ ] Product Owner asaccepted the User Story (demo of functionality completed) or waived the privilege.
 - [ ] All acceptance criteria are met.
-- [ ] I've added or updated relevant docs in the docs/ directory.
+- [ ] User Story has been updated (if needed) to match changes made during the development process.
+- [ ] I've added or updated relevant docs in the docs/ directory
+- [ ] QA has tested the additions and all found issues are triaged
```

Comment:
This one is a bit problematic, since QA usually only tests _after_ we've merged the PR, when it hits the stage env. And since they're part of the flow of the cards anyway (i.e. they'll pass through QA before getting to Done), I think it's probably fine to not have that in this checklist.

Reply:
I agree, it would be a big change to get QA to test PRs. Maybe part of merging should be preparing for stage deployment?

```suggestion
- [ ] User Story has been updated (if needed) with suggestions for QA when this PR is deployed to stage.
```